### PR TITLE
Update for core/formula split

### PR DESCRIPTION
**Pull request by apjanke**
Fix `brew-boneyard` to work with homebrew/core repo.
Update references in README.

`brew-boneyard` may be superseded by https://github.com/Homebrew/brew/pull/53, but may as well fix it in the mean time. Especially so you can run them side by side to compare results.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+## Installation
 
 ```
 brew tap homebrew/dev-tools
 ```
 
-## Add new tools
+## Adding new tools
 
-Read [Homebrew document](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/External-Commands.md) to see how to create Homebrew external commands.
```
Comment (xu-cheng) [inline README.md]: We probably should check all of official taps' README.md/CONTRIBUTION.md/etc.

File: cmd/brew-boneyard.rb [hunk 2]
```diff
@@ -42,7 +42,7 @@
   File.open(migrations, "w") do |f|
     f.write first_line
     m = mlines.slice(1..-2)
-    m << "  \"#{name}\" => \"homebrew/boneyard\",\n"
+    m << "  \"#{name}\": \"homebrew/boneyard\",\n"
```
Comment (xu-cheng) [inline cmd/brew-boneyard.rb]: Will this cause json syntax error if it's the last line.

**Comment by xu-cheng** [comment]
:+1:

File: cmd/brew-boneyard.rb (hunk 2 above)
Comment (apjanke) [inline cmd/brew-boneyard.rb]: Ah, yep. Will fix.

Comment (apjanke) [inline cmd/brew-boneyard.rb]: Amended to fix.

You can see the result at https://github.com/Homebrew/homebrew-core/pull/271, which I produced with the updated `brew-boneyard`. No trailing comma, and `json_pp` is happy parsing that `tap_migrations.json` file.

File: README.md (hunk 1 above)
Comment (apjanke) [inline README.md]: I added them to the list in the tracking issue: https://github.com/Homebrew/legacy-homebrew/issues/50650

File: cmd/brew-boneyard.rb [hunk 3]
```diff
@@ -1,29 +1,29 @@
 #! /usr/bin/env ruby
 # -*- coding: UTF-8 -*-
 #
-# Description: boneyard a formula.
-#   It assumes you have a personal fork of both the Homebrew core and the
+# Description: boneyard a formula from homebrew/core.
+#   It assumes you have a personal fork of both the homebrew/core and the
 #   boneyard tap. It creates a new branch on both repos; remove the formula in
```
Comment (ilovezfs) [inline cmd/brew-boneyard.rb]: remove, add, commit, push, open
-> removes, adds, commits, pushes, opens

also probably better to ; -> ,

**Comment by ilovezfs** [comment]
```
== brew-boneyard.rb ==
C: 14:  1: Always use raise to signal exceptions.
C: 33:  3: Always use raise to signal exceptions.
C: 34:  3: Always use raise to signal exceptions.

1 file inspected, 3 offenses detected

```

File: cmd/brew-boneyard.rb (hunk 3 above)
Comment (apjanke) [inline cmd/brew-boneyard.rb]: Amended to fix grammar and convert to a bullet list.

File: cmd/brew-boneyard.rb [hunk 4]
```diff
@@ -1,51 +1,55 @@
...
 # -*- coding: UTF-8 -*-
 #
-# Description: boneyard a formula.
-#   It assumes you have a personal fork of both the Homebrew core and the
-#   boneyard tap. It creates a new branch on both repos; remove the formula in
-#   the core and add it to tap_migrations.rb; add it in the boneyard repo;
-#   commit and push in both; and open both PR pages in your browser.
+# Description: boneyard a formula from homebrew/core.
+#   This assumes you have a personal fork of both the homebrew/core and
+#   homebrew/boneyard tap repos. It:
```
Comment (ilovezfs) [inline cmd/brew-boneyard.rb]: don't need the : after It

File: README.md [hunk 5]
```diff
@@ -1,17 +1,17 @@
...
+Read the ["External Commands" Homebrew document](https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/External-Commands.md) to see how to create Homebrew external commands.
 
-As a good practice, adding information on usage and origin as ruby comment is recommended.
+As a good practice, adding information on usage and origin as ruby comments is recommended.
```
Comment (ilovezfs) [inline README.md]: Maybe just drop "as a good practice" or "best practice is ..." and drop the "is recommended."

File: cmd/brew-boneyard.rb (hunk 4 above)
Comment (apjanke) [inline cmd/brew-boneyard.rb]: I think it's appropriate here since it's introducing a list.

Comment (ilovezfs) [inline cmd/brew-boneyard.rb]: Sure, if it were "It does the following:"

File: README.md (hunk 5 above)
Comment (apjanke) [inline README.md]: Let's defer additional minor style changes to the helptext til later. This fixes it for core/formula split, and it should have an upcoming PR to convert the comment block to the new subcommand help syntax and style anyway.

**Comment by ilovezfs** [comment]
:+1:

File: cmd/brew-boneyard.rb (hunk 4 above)
Comment (apjanke) [inline cmd/brew-boneyard.rb]: I think that's implicit. And if you just remove the ":" then you have a sentence fragment consisting of "it". Looks sloppy.

**Pull request closed without merging**